src/runtime/pickle_runner.ts

```diff
@@ -50,6 +53,7 @@ export default class PickleRunner {
   private readonly pickle: messages.IPickle
   private readonly maxAttempts: number
   private result: messages.TestStepFinished.ITestStepResult
+  private stepResult: messages.TestStepFinished.ITestStepResult
```

Comment:
I disagree with having this as a class variable.  I think it can just be passed into the `runStepHook` function

Reply:
Updated to pass directly into the function